ui/user/src/routes/v2/admin/chat-configuration/+page.svelte

```diff
@@ -264,17 +264,19 @@
 									>
 										Set as Default
 									</button>
-									<button
-										class="menu-button"
-										onclick={() => {
-											if (!baseAgent) return;
-											baseAgent.allowedModels = baseAgent.allowedModels?.filter(
-												(modelName) => modelName !== d.name
-											);
-										}}
-									>
-										Remove
-									</button>
+									{#if !d.isDefault}
```

Comment:
@ivyjeong13 I added so that we don't allow user to remove a model that has been set as default to prevent weird behavior like https://github.com/obot-platform/obot/issues/3593